app/models/katello/rhsm_fact_parser.rb

```diff
@@ -79,6 +79,22 @@ def boot_timestamp
       facts['proc_stat.btime']&.to_i
     end
 
+    def virtual
+      facts['virt.is_guest']
+    end
+
+    def ram
+      facts['memory.memtotal'].to_i / 1024 if facts['memory.memtotal']
```

Comment:
I'm still seeing RAM as infinity, looks like due to facts['memory.memtotal'] being nil. 
```
irb(main):014:0> Host.find(2).facts["memory.memtotal"]
=> nil
```

I'm seeing this as `memory::memtotal` on my machine:
```
irb(main):013:0> Host.find(2).facts["memory::memtotal"]
=> "990712"
```